docs/src/ScrollableContainer.doc.js

```diff
@@ -391,195 +400,195 @@ function ScrollableContainerExample() {
               </Box>
             }
           >
-            <ScrollableContainer>
+            <Box
+              display="flex"
+              justifyContent="center"
+            >
               <Box
```

Comment:
This should use Flex to obviate needing to set padding on all the children, as well as the negative margins on this parent (use Flex.Item for the children to set `flex="grow"`)

Reply:
Let's update this in a separate PR, along with Layouts.

Reply:
Sure, sounds good 👍